app/src/ui/accessibility/aria-live-container.tsx

```diff
@@ -1,7 +1,23 @@
 import { debounce } from 'lodash'
 import React, { Component } from 'react'
 
+export interface IAccessibleMessage {
```

Comment:
Took me a while to understand that this is just some kind of auxiliar structure/interface and nothing actually used by the `AriaLiveContainer`.

Could we add a comment describing it and maybe even move it to its own file?